.github/workflows/prune-preview-deploys.yml

```diff
@@ -31,6 +31,12 @@ jobs:
       - name: Setup Fly IO
         uses: superfly/flyctl-actions/setup-flyctl@master
 
+      - name: Configure AWS credentials using OIDC
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::571248596162:role/huppy-bot
```

Comment:
We only prune preview, so this is hard coded (for deploys we have to switch between different profiles).